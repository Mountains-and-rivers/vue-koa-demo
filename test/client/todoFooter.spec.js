import { shallowMount } from '@vue/test-utils'
import Todofooter from '@/components/Todofooter'

describe('Todofooter.vue', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallowMount(Todofooter)
  })

  test('render text footer', () => {
    expect(wrapper.html()).toContain('<footer>vue-koa-todos by <span>BreezyMelon</span></footer>')
  })

  test('todofooter.vue snapshot test', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
