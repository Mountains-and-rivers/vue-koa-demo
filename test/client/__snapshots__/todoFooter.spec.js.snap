// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todofooter.vue todofooter.vue snapshot test 1`] = `
<div
  class="todo-footer"
>
  <footer>
    vue-koa-todos by 
    <span>
      BreezyMelon
    </span>
  </footer>
</div>
`;
